pkg/apis/devopsconsole/v1alpha1/component_types.go

```diff
@@ -0,0 +1,53 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// ComponentSpec defines the desired state of Component
+// +k8s:openapi-gen=true
+type ComponentSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	App        string `json:"app"`
+	Codebase   string `json:"codebase"`
+	Buildtype  string `json:"nodejs"`
```

Comment:
@sbose78  Shouldn't It supposed to be `build_type` ?

Reply:
Yes, right!
In your devfile PoC , change it and use this branch :)

Reply:
Yes, right!
In your devfile PoC , change it and use this branch :)

Reply:
Is there a reason why you don't use our [ComponentSpec](https://github.com/snowdrop/component-operator/blob/master/pkg/apis/component/v1alpha1/component_types.go) enriched with your needs ? @sbose78

Reply:
@cmoulliard , Corinne is on that - she is evaluating it and getting something working. This is a PoC PR.